### PR TITLE
chore: changed link to "here"

### PR DESCRIPTION
**Pull request by ErikSin**
# Updated link for gitbooks to be less verbose
In Experimental Features>Background Maps, the link to the online gitbooks page has been updated to the correct link. The url has also been hidden and now the user can click on "here"

image

### CODE REVIEW

**Review by achou11 — COMMENTED** [review]
The wording here is awkward from a grammar standpoint (most notably the `here` after a colon). Would reword to something like 

> See the [online documentation] to learn more about generating map files.

(where `[]` is the linked text)

**Comment by achou11** [comment]
Would point this to `develop` branch (see Slack)

**Comment by ErikSin** [comment]
> The wording here is awkward from a grammar standpoint (most notably the `here` after a colon). Would reword to something like
> 
> > See the [online documentation] to learn more about generating map files.
> 
> (where `[]` is the linked text)

This is based off what Emilie Reiser's [Feedback](https://www.notion.so/digidem/Tech-Priorities-a8bb127b4ba94322ae2a8cd75477e949?p=1eed67e3a0be4319b49b955f9992cbc0&pm=s). Maybe bring it up in the notion page?

**Comment by achou11** [comment]
> This is based off what Emilie Reiser's [Feedback](https://www.notion.so/digidem/Tech-Priorities-a8bb127b4ba94322ae2a8cd75477e949?p=1eed67e3a0be4319b49b955f9992cbc0&pm=s). Maybe bring it up in the notion page?

ah I see. will bring it up but in that case, let's not make this a blocker for this PR 😄

**Comment by achou11** [comment]
before mergining, can you remove the first commit in this PR (the release commit)? looks like switching target branches carried it over
